### PR TITLE
Add compat prefixed keys to recommended alternate features

### PR DESCRIPTION
**Pull request by jamesnw**
These keys are a bit odd- they are in the Compat standard, meaning they only exist because they were essentially a standard, but you shouldn't use them. I don't think there is a clear reason to have a separate feature for these, but I could be convinced. The resolution addition is very straightforward, but the addition of `-webkit-transform-3d` to `supports` is a bit odd, even though it is the recommended way of doing the same thing.

### CODE REVIEW

**Comment by ddbeck** [comment]
This situation is a bit messy. It implicates feature definitions and also BCD's structure and deprecated flag. See https://github.com/web-platform-dx/web-features/issues/86 and https://github.com/mdn/browser-compat-data/issues/22958 for past discussion.

> you shouldn't use them

My preference is that we ought to mark such things as deprecated in BCD (or discouraged here, once we sort out https://github.com/web-platform-dx/web-features/issues/311).

Looking over the spec, I expect that's a bit of work, since the text is a little equivocal. I think we'd have to check each one and find out if an unprefixed equivalent exists and only mark those as discouraged. That sounds hard and time consuming. So maybe we just need to set these things aside so we can clean them up later.

Perhaps something we can do here and now is distinguish these items from their parent feature, but keep them closely together. Maybe something like…

```yml
# resolution-compat.yml
name: resolution media query (compatibility prefixes)
description: The `-webkit-device-pixel-ratio`, `-webkit-min-device-pixel-ratio`, and `-webkit-max-device-pixel-ratio` CSS media queries are standardized compatibility alternatives to `resolution` media queries.
```

**Comment by jamesnw** [comment]
@ddbeck That makes sense to me. I split out `-webkit-transform-3d` from supports as well- does that still make sense, even if it's not as 1:1?

**Comment by ddbeck** [comment]
Yeah, I think this looks OK. @Elchi3 would you have a look at this too?

File: features/resolution-compat.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: resolution media query (compatibility prefixes)
```
Comment (Elchi3) [inline features/resolution-compat.yml:1]:
```suggestion
name: "@media resolution (compatibility prefixes)"
```

File: features/supports-compat.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: "@supports (compatibility prefix)"
+description: The `-webkit-transform-3d` CSS media query is a standardized compatibility alternative to `@supports (transform-style)`.
```
Comment (Elchi3) [inline features/supports-compat.yml:2]:
```suggestion
description: The `@supports (-webkit-transform-3d)` CSS media query is a standardized compatibility alternative to `@supports (transform-style)`.
```

**Review by Elchi3 — APPROVED** [review]
Looks good to me. Two optional suggestions that would make things more consistent to my eyes.

File: features/resolution-compat.yml (hunk 1 above)
Comment (ddbeck) [reply features/resolution-compat.yml:1]: "resolution media query (compatibility prefixes)" is modeled after `resolution.yml`'s name. If this changes (which is a reasonable suggestion), we'd also need to change the other feature.

(I'm a tiny bit worried about having a bunch of `@media` features but this doesn't strike me as a "web" prefix situation.)

**Pull request merged**